File: Examples/CSharp/AsposePDF/Bookmarks/DeleteAllBookmarks.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.Bookmarks
{
    public class DeleteAllBookmarks
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_Bookmarks();

            //open document
            Document pdfDocument = new Document(dataDir + "DeleteAllBookmarks.pdf");

            //delete all bookmarks
            pdfDocument.Outlines.Delete();

            //save updated file
            pdfDocument.Save(dataDir + "DeleteAllBookmarks_out.pdf");

        }
    }
}

File: Examples/VisualBasic/AsposePDF/Bookmarks/DeleteAllBookmarks.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf
Namespace AsposePDF.Bookmarks
    Public Class DeleteAllBookmarks
        Public Shared Sub Run()
            ' ExStart:DeleteAllBookmarks
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_Bookmarks()

            ' Open document
            Dim pdfDocument As New Document(dataDir & Convert.ToString("DeleteAllBookmarks.pdf"))

            ' Delete all bookmarks
            pdfDocument.Outlines.Delete()

            dataDir = dataDir & Convert.ToString("DeleteAllBookmarks_out_.pdf")
            ' Save updated file
            pdfDocument.Save(dataDir)
            ' ExEnd:DeleteAllBookmarks
            Console.WriteLine(Convert.ToString(vbLf & "All bookmarks deleted successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace